scripts/upgradeChecks.js

```diff
@@ -0,0 +1,142 @@
+const config = require('../config.json');
+let network = 'mainnet'
+
+async function main() {
+    const addr = config.addresses;
+
+    const netinfo = await ethers.provider.getNetwork();
+    network = netinfo.name;
+    if (network === "unknown")
```

Comment:
Change this to 

```
 if (network === "unknown"){
network = 'mainnet';
 // put the rest of the code here
}
```

Reply:
I have a question here, do you mean that put the same code here for mainnet?
In my opinion, it's not good we have duplicated code for mainnet, binance and polygon.
so, how about to use if/else for only places where needs a special process for a specific network?
Or we can define configs for each networks, it will makes generalized code.
Forexample, in deploy.js and config.json
```
	const wrapper = await deployWithProxy(
        Wrapper, 
        OwnableProxy, 
        'WrapAndUnWrap',
		addr.tokens.WETH[network],
		addr.swaps.uniswap[network],
		addr.swaps.uniswapFactory[network],
		addr.tokens.DAI[network],
		addr.tokens.USDT[network],
		addr.tokens.USDC[network]
    );
```
The more generalized code, easier to understand and more clean.
 What do you think?

Reply:
@softskillpro I agree, please implement it this way :)